src/osp/Resource/ImporterData.h

```diff
@@ -83,7 +84,12 @@ struct ImporterData
 struct Prefabs
 {
     // [prefab Id][object]
-    lgrn::IntArrayMultiMap<int, int>    m_prefabs;
+    lgrn::IntArrayMultiMap<int, int>    m_prefabs{};
+    lgrn::IntArrayMultiMap<int, int>    m_prefabParents{};
+    std::vector<std::string_view>       m_prefabNames;
```

Comment:
storing string_views as member variables sounds like a dangling pointer waiting to happen.

Can you document in the code the lifetime of these?